src/UofIQualys/functions/public/Stop-QualysScan.ps1

```diff
@@ -0,0 +1,36 @@
+﻿<#
+.Synopsis
+    Stop a vulnerability scan in the user’s account.
+.DESCRIPTION
+    Stop a vulnerability scan in the user’s account.
+.PARAMETER ScanRef
+    The target ScanRef for a vulnerability scan.
+.EXAMPLE
+    Stop-QualysScan -ScanRef 'scan/1633304415.63272'
+#>
+function Stop-QualysScan{
+    [CmdletBinding(SupportsShouldProcess)]
+    param (
+        [Alias('scan_ref')]
+        [String]$ScanRef
+    )
+
+    process{
+        if ($PSCmdlet.ShouldProcess("$($ScanRef)")){
```

Comment:
Using both arguments for TARGET and OPERATION make the output a little more clear
```suggestion
        if ($PSCmdlet.ShouldProcess("$($ScanRef)","Stop")){
```

Reply:
This will give the output "Performing operation 'Stop' on target $ScanRef"